modules/configuring-pods-static-ip.adoc

```diff
@@ -10,7 +10,7 @@ The following example provisions a pod with a static IP address.
 
 [NOTE]
 ====
-* You can only specify the IP address for a pod's secondary network attachment for layer 2 attachments.
+* For a pod that exists in the additional OVN-Kubernetes network, you can specify a static IP address for the pod only if the additional network attachment uses a layer 2 or a localnet topology framework for its network configuration.
```

Comment:
@dfitzmau For a pod to be attached to additional OVN-Kubernetes network , can have static IP address assigned if the additional network uses layer 2 or localnet topology for its network configuration suffices.

Not sure about framework. Also do you want to clarify the additional configuration is secondary now that we have UDN.

Reply:
I updated the bullet point to state:

> * For a pod to attach to an additional OVN-Kubernetes network, the additional OVN-Kubernetes network must use a layer 2 or localnet topology for its network configuration, so that you can assign a static IP address to the pod.

I checked the [UDN docs|https://docs.openshift.com/container-platform/4.18/networking/multiple_networks/primary_networks/about-user-defined-networks.html#nw-cudn-cr_about-user-defined-networks] and I see the following statement 

![Screenshot From 2025-03-10 16-22-24](https://github.com/user-attachments/assets/57bb3046-0e46-499f-8d19-e724b1494b14)

Is it worthwhile mentioning "UDN" in the "Configuring pods with a static IP address" section?

Reply:
I misspoke NAD can be secondary without UDN as well, without UDN - default network + secondary interface. Just wanted to highlight specifically for Layer 2, only if it is secondary pods can be assigned static IPs. So you may refer to it as secondary without UDN, it will suffice.